haskellfoundation/error-message-index
# adds OS controlled dark mode

## Changes
- removes all theme-dependent colors from default css
- adds _theme.css_ with a dark and light color palette that respects the OS theme settings

## How to Test
- run site locally
    - `stack run -- site watch`
    - navigate to site: `http://localhost:8000/`
- Chrome/Safari (macOs)
    - go to settings > appearance
        - toggle between Light and Dark mode and see that the page changes color modes
- Firefox (macOs)
    - open developer tools and toggle between sun and moon icon (see ss)
 
<img width="300" src="https://user-images.githubusercontent.com/13749324/220777513-93d8b521-4af6-4d48-8ed1-95ac42d28892.png">



## Related Issue
- https://github.com/haskellfoundation/error-message-index/issues/353

### Screen Shot
<img width="1774" alt="Screenshot 2023-02-21 at 7 15 37 PM" src="https://user-images.githubusercontent.com/13749324/220488059-bf87b482-7802-42ed-a8db-b88bd32b717f.png">
